src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -82,7 +82,7 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
       datasetId = props.getString("identifier.fedora", "")
       ddmOut <- configuration.ddmTransformer.transform(ddmIn, datasetId)
       _ = Provenance(ddmIn, ddmOut, s"${ getClass.getSimpleName } ${ configuration.version }")
-        .foreach(s => logger.info(s))
+        .foreach(xml => (bagDir / "metadata" / "prov.xml").writeText(xml.serialize))
```

Comment:
not touched by unit tests

Reply:
We might revert this line so merge does not have to wait for updates of schemas and bag validation.